warehouse/admin/static/css/admin.scss

```diff
@@ -28,3 +28,11 @@
 .table th {
   max-width: 30em;
 }
+
+.preserve-line-breaks {
+  white-space: pre-line;
+}
+
+.direct-chat-messages-full-height {
+  height: unset;
+}
```

Comment:
Considering the little volume of modifications to admin styles, this is likely fine to use without namespacing under `direct-chat-messages`, but if we ever revisit our admin UI CSS we should consider these then.

Reply:
96aacc53b52d16151c2c9564714ef3350904b292